Dockerfile

```diff
@@ -21,8 +21,11 @@ RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -ldflags="-w -
 FROM gcr.io/distroless/base
 ENV DISTRO="debian"
 ENV GOARCH="amd64"
+ENV SERVICE_ADDR="meshery-istio"
+ENV MESHERY_SERVER="http://meshery:9081"
```

Comment:
@kumarabd what does this mean for the adapter when deployed in a Kubernetes cluster?

Reply:
Just the name of the service in the grpc definition.

Reply:
Nothing for the kubernetes cluster